packages/flutter/lib/src/gestures/long_press.dart

```diff
@@ -19,20 +22,36 @@ class LongPressGestureRecognizer extends PrimaryPointerGestureRecognizer {
   /// Consider assigning the [onLongPress] callback after creating this object.
   LongPressGestureRecognizer({ Object debugOwner }) : super(deadline: kLongPressTimeout, debugOwner: debugOwner);
 
+  //is a flag which is used to determine if the longPress action happened
```

Comment:
you can omit this comment, i think it's clear what the variable does and the comment doesn't really add much.